sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -393,6 +393,45 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * This function goes through the input once to determine the input schema. To avoid going
    * through the entire data once, specify the schema explicitly using [[schema]].
    *
+   * You can set the following CSV-specific options to deal with CSV files:
+   * <li>`sep` or `delimiter` (default `,`): sets the single character as a delimiter for each
+   * field and value.</li>
+   * <li>`encoding` or `charset` (default `UTF-8`): decodes the CSV files by the given encoding
```

Comment:
BTW a really good reference for these documentation options is R: https://stat.ethz.ch/R-manual/R-devel/library/utils/html/read.table.html